qbraid/transpiler/conversions/openqasm3/__init__.py

```
@@ -26,10 +26,13 @@
    openqasm3_to_cudaq
+   openqasm3_to_pyquil
 
 """
+
 from .openqasm3_to_cudaq import openqasm3_to_cudaq
 from .openqasm3_to_ionq import openqasm3_to_ionq
+from .openqasm3_to_pyquil import openqasm3_to_pyquil
 from .openqasm3_to_qasm3 import openqasm3_to_qasm3
 
-__all__ = ["openqasm3_to_qasm3", "openqasm3_to_ionq", "openqasm3_to_cudaq"]
+__all__ = ["openqasm3_to_qasm3", "openqasm3_to_ionq", "openqasm3_to_cudaq", "openqasm3_to_pyquil"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)